app/controllers/api/dishes_controller.rb

```diff
@@ -48,5 +50,13 @@ def find_dish
     def dish_params
       params.permit(:user_id, :name, :price)
     end
+
+    def user_not_authorized
+      render json: user_not_authorized_error_message, status: :unauthorized
+    end
+
+    def user_not_authorized_error_message
+      {error: {dish: {message: 'Debt too large', limit: Dish::MAX_DEBT}}}
+    end
```

Comment:
1) Why a separate method for a simple JSON? Is it used anywhere else? 
2) `message` suggests a string or a text, and this method returns a hash.
3) `user_not_authorized_error_message` is not very telling because in this instance the reason they're not authorized is very specific. If anything, I'd go with `debt_too_large_message` method name, but see 1 and 2.